Library/SPIClient.m

```diff
@@ -1520,19 +1520,27 @@ - (void)transactionMonitoring {
 }
 
 - (void)handlePrintingResponse:(SPIMessage *)m {
-    [_delegate printingResponse:m];
+    if([_delegate conformsToProtocol:@protocol(SPIDelegate)] && [_delegate respondsToSelector:@selector(printingResponse:)]) {
```

Comment:
Traditionally in ObjC, people only check the respondsToSelector, but no big deal. ```suggestion
    if([_delegate respondsToSelector:@selector(printingResponse:)]) {
```

Reply:
I applied it. Thanks.